meson.build

```diff
@@ -25,41 +62,70 @@ gst_audio_dep = dependency('gstreamer-audio-' + gst_api_verision)
 gst_app_dep = dependency('gstreamer-app-' + gst_api_verision)
 gst_check_dep = dependency('gstreamer-check-' + gst_api_verision)
 
-cc = meson.get_compiler('c')
-cxx = meson.get_compiler('cpp')
+libm_dep = cc.find_library('m') # cmath library
+libdl_dep = cc.find_library('dl') # DL library
+libpthread_dep = cxx.find_library('pthread') # pthread for tensorflow-lite
 
-nnstreamer_deps = [
+nnstreamer_base_deps = [
   glib_dep,
   gst_dep,
   gst_base_dep,
   gst_controller_dep,
   gst_video_dep,
   gst_audio_dep,
-  # required 'false' mean is used only when needed.
-  cc.find_library('m', required: false), # cmath library
-  cc.find_library('dl', required: false), # DL library
-  cxx.find_library('pthread', required: false) # pthread for tensor_filter_tensorflow_lite
+  libm_dep,
+  libdl_dep,
+  libpthread_dep
 ]
 
-nnstreamer_inc = include_directories('gst/nnstreamer')
-nnstreamer_base_args = []
+# For extra dependencies (subplugins, filters)
+nnstreamer_extra_deps = []
 
 # Orc
 have_orcc = false
-orcc_args = []
 
-orc_dep = dependency('orc-0.4', version: '>= 0.4.24', required: true)
-orcc = find_program('orcc', required: true)
+orc_dep = dependency('orc-0.4', version: '>= 0.4.17', required: false)
+orcc = find_program('orcc', required: false)
 if orc_dep.found() and orcc.found()
   have_orcc = true
   orcc_args = [orcc, '--include', 'glib.h']
-  nnstreamer_deps += [orc_dep]
+  nnstreamer_base_deps += [orc_dep]
   nnstreamer_base_args += ['-DHAVE_ORC=1']
 else
   message('Cannot find orc library')
   nnstreamer_base_args += ['-DDISABLE_ORC=1']
 endif
 
+# Tensorflow
+have_tensorflow = false
+
+if get_option('ENABLE_TENSORFLOW')
+  tf_dep = dependency('tensorflow', required: false)
+  protobuf_dep = dependency('protobuf', version: '>= 3.4.0', required: false)
```

Comment:
set required: true, if ENABLE_TENSORFLOW is true, both are required.

Reply:
protobuf is not working with this BECAUSE debian packaging of https://github.com/nnsuite/tizenport-protobuf is incomplete. *.install files are missing from that repo; thus it's not working properly.